src/Components/TheoryPage.js

```diff
@@ -0,0 +1,989 @@
+              <View style={styles.modalViewTheory}>
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(0);
+                    setTheorySubject('Vertical Stack');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Vertical Stack</Text>
+                </TouchableHighlight>
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(1);
+                    setTheorySubject('Horizontal Stack');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Horizontal Stack</Text>
+                </TouchableHighlight>
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(2);
+                    setTheorySubject('Endzone Offense');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Endzone Offense</Text>
+                </TouchableHighlight>
+
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(3);
+                    setTheorySubject('Handling');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Handling</Text>
+                </TouchableHighlight>
+
+                <TouchableHighlight
+                  style={styles.returnButton}
+                  onPress={() => {
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.textStyle}>Return</Text>
+                </TouchableHighlight>
+              </View>
+            </View>
+          </Modal>
+        </View>
+        <View style={styles.displayTheory}>
+          <TouchableHighlight
+            style={styles.subjectModal}
+            onPress={() => {
+              setModalVisible(true);
+            }}
+          >
+            <View>
+              <View style={styles.dropdown}>
+                <Text style={{ ...styles.textStyle, color: theme.COLOR_PRIMARY }}>{theorySubject}</Text>
+                <MaterialCommunityIcons name="chevron-down" color={theme.COLOR_PRIMARY} size={26} />
+              </View>
+              <View style={styles.lines} />
+            </View>
+          </TouchableHighlight>
+        </View>
+        <View style={styles.displayTheory}>
+          <View>{displayTheory()}</View>
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default function TheoryPage() {
+  return (
+    <Tab.Navigator
```

Comment:
I'm afraid another bottom tab navigator could be misleading for the user. We need to test this code updated with the master branch to figure it out

Don't you think we could add a "Theory" tab on the home page, with 3 images as we did for fitness and frisbee?
(We would probably need to rename Frisbee to Training)

Reply:
I am going to have a try